playground-common/src/Gist.hs

```diff
@@ -142,12 +141,11 @@ instance FromJSON Gist where
             _gistGitPushUrl <- o .: "git_push_url"
             _gistHtmlUrl <- o .: "html_url"
             _gistOwner <- o .: "owner"
-            _gistFiles <-
-                Map.elems <$> ((o .: "files") :: Parser (Map String GistFile))
+            _gistFiles <- (o .: "files")
             _gistTruncated <- o .: "truncated"
             _gistCreatedAt <- o .: "created_at"
             _gistUpdatedAt <- o .: "updated_at"
-            _gistDescription <- o .: "description"
+            _gistDescription <- o .:? "description" .!= ""
```

Comment:
this is a backwards compatibility thing, every playground gist should have a description but if there is some weird gist lying around (which there was for me) we don't want it to break everything, the simplest way is to just blank it out here rather than having anything more serious.

Reply:
Ah, okay. That makes sense. Add a code comment maybe?

Reply:
done.